### PR TITLE
Pawnch Update 2.5: I swear this will be the last one for a while. + some slight fixes

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Firstly, what I fixed: Hugs of the North Star are no longer nodrop, ashen arrows are actually called that in the crafting menu (I'm touching it here while I'm touching crafting), flag checks and knockdown overrides were added into various moves for martial arts.

**Pugilists and Armwraps:**
Pugilists can now disarm you more easily, and you can't disarm them as easily. I don't like RNG disarm but kev likes it so eeeeeeh. This means pugilist gloves can serve as a means of protecting yourself from RNG! Also they get a stamina discount.

Pugilists have a standard flat 10% miss chance regardless of stamina or brute damage. This is basically what old punches were like, while everyone else gets to suffer under dreaded dice rolling growing ever more unfavorable as they throw punches.

Armwraps can be crafted. You can even make magical ones. Yay!

Armbands of the Everlasting Will (the chaplain ones) now are a +2.

**Martial Arts**
The four main martial arts now have some updates and added in functionality for stun thresholds being met that they otherwise lost when they gained martial arts. _**Most if not all stamina damage dealt by martial arts has been increased.**_ This is because it was really damn low before given stamina regeneration makes it somewhat mandatory for stamina damage to be dealt out in bursts. This was mostly touching the flat damage and not the rng values, which are not increased by targets in and out of combat mode.

**Edit:** Also martial artists are spending stamina when they disarm as they should. Missed this in my last update.

**CQC**
Disarm act isn't a weird version of the RNG disarm anymore, it's a big brother version to the Krav Maga disarm jab. You do a pretty okayish amount of stamina damage, and if you roll a stun punch, you dizzy them and cause them to drop their weapons. There is no miss chance for this. This hasn't changed any other functionality to disarm. This is what leads up to Consecutive CQC, so it being a sequence of strikes makes sense. This also makes that move actually worth using. You're still not going to use it because grab harm exists but we'll get onto CQC's various issues at a later date.

**Rising Bass**
The moves Repulse Punch and Shoulder Flip have had their combos reduced by a few moves and an altered move sequence. Rising Bass also has a disarm act override that is suitably fitting for being a rakish arsehole, which Rising Bass seems to promote.

Repulse Punch has a lighter knockdown and a little more stamina damage. You can perform a Repulse Punch by Harm Disarm Disarm (think Crashing Wave Kick but two more disarms). This move still has an inbuilt cooldown by the way, so don't worry too much about it being spammed.

Shoulder Flip works even on people who are already on the floor. This move also does less actual brute damage but does a fairly respectable amount of stamina damage. If you roll damage over the stun threshold, however, it also puts your target to sleep. This move used to do this already, but I had to make it slightly less reliable given I shaved off a hit to complete it. The move sequence is now Disarm Harm Disarm Harm Grab (the old move sequence was Grab Harm Disarm Grab Harm Harm which I don't think I can ever see anyone pulling off especially given they had to be standing as well)

Disarms are now a trip if you roll above the stun threshold, or a punch to the stomach for some stamina damage. It causes disgust. Have fun.

**Sleeping Carp**
Harm punches can floor people if you get a stun punch. It's not that big a deal but it can make rolling high for damage a little more rewarding. **_This does not affect Gnashing Teeth._**

**Krav Maga**
Disarm act has had it's RNG chance to disarm you changed to a stun punch. Should prevent too much disarm spam.

## Why It's Good For The Game

I cannot stop touching punch code aaaaah.

## Changelog
:cl:
add: You can now craft armwraps!
balance: Pugilists disarm you more easily and are harder to disarm. They also get a discount on disarm.
balance: Pugilists only suffer a flat 10% chance to miss you. It's just like old punches! Kinda.
balance: Chaplain's armbands are a +2, up from a +1!
balance: Martial artists spend stamina when they disarm.
balance: Rising Bass had several moves shortened and made stronger. Has a disarm override attack which does stamina damage and trips people on a disarm stun punch.
balance: CQC had it's disarm move altered to be a stronger version of Krav Maga's. Dizzies and disarms on a disarm stun punch or just does some stamina damage and brute damage.
balance: Sleeping Carp can punch you to the floor on a harm stun punch.
fix: Hugs of the Northstar are no longer nodrop.
fix: Adding in some overrides and proper flag checks for martial arts.
fix: Stun thresholding stops disarm spams at extremely high stamina loss.
tweak: Ashen Arrows are actually called Ashen Arrows in the crafting menu.
/:cl:

### CODE REVIEW

File: code/datums/components/crafting/recipes/recipes_clothing.dm [hunk 1]
```diff
@@ -39,6 +39,30 @@
...
+	name = "armwraps of mighty fists"
+	result = /obj/item/clothing/gloves/fingerless/pugilist/magic
+	time = 300
+	tools = list(TOOL_WIRECUTTER, /obj/item/book/codex_gigas, /obj/item/clothing/head/wizard, /obj/item/clothing/suit/wizrobe)
+	reqs = list(/obj/item/stack/sheet/cloth = 2,
+				/obj/item/stack/sheet/leather = 2,
+				/obj/item/stack/sheet/durathread = 2,
+				/datum/reagent/consumable/ethanol/sake = 100,
+				/datum/reagent/consumable/ethanol/wizz_fizz = 100,
+				/obj/item/stack/sheet/sinew = 1,
```
Comment (Trilbyspaceclone) [inline code/datums/components/crafting/recipes/recipes_clothing.dm:63]: maybe not sinew and 50 sheets of gold
Silk and 10 sheets of gold with 5 sheets of silver would be better?

Comment (necromanceranne) [reply code/datums/components/crafting/recipes/recipes_clothing.dm:63]: 50 gold sheets is 1000 units of gold reagent. Thats the best approximation to 1000 gold pieces I could use for an elaborate 3.X dnd joke without making someone quite literally go and press 1000 gold coins from the horrifically archaic coin press. 1000 gold pieces was the price of a +1 magic weapon you make yourself. Technically this is a flawed joke as amulets of mighty fists, which is what this item is a reference too, is 1500 gold pieces due to dumb pricing shit, but you get the idea. Nobody is going to call me out on an extremely obtuse joke.

Also the sinew isn't that hard to acquire. The idea is that it uses rare and interesting materials as components. Sinew means you need to either hit up xenobio or a miner, wizz fizz and sake the bartender, durathread the botanists (as well as the cloth and leather probably), and the gold from hopefully mining doing their job. The gigas codex means you need a curator to help, and the robes and hat from going to the theatre for the fakes. Magical items were always the subject of elaborate bullshit and this gives some reasonable deniability to whether it's magic or the materials used to make the item making it distinctly durable and the enhancement a placebo. You just THINK it should hit harder, so you hold back a little less.

Thank you for coming to my TEDTalk where I explain 'the joke that nobody asked for'.

**Comment by timothyteakettle** [comment]
if you're touching punch code, please hit mush punch with a nerf because it's a silent punch that can be stacked and does incredibly high damage

**Comment by necromanceranne** [comment]
It's because someone thought adding a race with a martial art was a good idea. Take it up with whoever enabled them as roundstart or remove it from their species.

**Review by silicons — APPROVED** [review]

**Pull request merged**